plugins/services/src/js/reducers/JSONMultiContainerReducers.js

```diff
@@ -24,10 +24,7 @@ module.exports = {
   environment,
   scaling,
   labels,
-  scheduling(
-    state = { residency: {}, placement: { constraints: [] } },
```

Comment:
Isn't this a problem in line 36 where we do residency.bind(this)?

Reply:
I think `residency` on line 36 comes from the import on line 24.

```js
import {
  JSONReducer as residency
} from "./serviceForm/JSONReducers/MultiContainerResidency";
```